modules/custom/openy_font/src/Form/FontSettingsForm.php

```diff
@@ -36,6 +36,17 @@ public function buildForm(array $form, FormStateInterface $form_state) {
 
     $form['#title'] = $this->t('OpenY Font Settings');
 
+    if (\Drupal::service('module_handler')->moduleExists('fontyourface')) {
```

Comment:
What happens when openy_font not anymore in the repo and disabled?
User still will be seeing this message, which is confusing. Could you make it better?

Reply:
Better could be if you check for both openy_font and fontyourface exist in a system
If openy_font exist, but disabled or doesn't exist - no need to display this message @kolesnikoff